MapboxDirections/MBRouteOptions.swift

```diff
@@ -201,126 +246,151 @@ open class RouteOptions: NSObject, NSSecureCoding {
         return "directions/v5/\(profileIdentifier.rawValue)/\(queryComponent).json"
     }
-    
+
     /**
      An array of directions query strings to include in the request URL.
      */
     internal var queries: [String] {
         return waypoints.map { "\($0.coordinate.longitude),\($0.coordinate.latitude)" }
     }
-    
+
     /**
      A Boolean value indicating whether the route includes a `ManeuverType.exitRoundabout` or `ManeuverType.exitRotary` step when traversing a roundabout or rotary, respectively.
-     
+
      If this option is set to `true`, a route that traverses a roundabout includes both a `ManeuverType.takeRoundabout` step and a `ManeuverType.exitRoundabout` step; likewise, a route that traverses a large, named roundabout includes both a `ManeuverType.takeRotary` step and a `ManeuverType.exitRotary` step. Otherwise, it only includes a `ManeuverType.takeRoundabout` or `ManeuverType.takeRotary` step. This option is set to `false` by default.
      */
     open var includesExitRoundaboutManeuver = false
-    
+
     /**
      The locale in which the routes’ instructions are written.
-     
+
      If you use MapboxDirections.swift with the Mapbox Directions API, this property affects the sentence contained within the `RouteStep.instructions` property, but it does not affect any road names contained in that property or other properties such as `RouteStep.name`.
-     
+
      The Directions API can provide instructions in [a number of languages](https://www.mapbox.com/api-documentation/#instructions-languages). Set this property to `Bundle.main.preferredLocalizations.first` or `Locale.autoupdatingCurrent` to match the application’s language or the system language, respectively.
-     
+
      By default, this property is set to `nil`, causing instructions to be written in the default language, English.
      */
     open var locale: Locale?
     
     /**
+     A Boolean value indicating whether each route step includes an array of `SpokenInstructions`.
+
+     If this option is set to true, the `RouteStep.instructionsSpokenAlongStep` property is set to an array of `SpokenInstructions`.
+     */
+    open var includesVoiceInstructions = false
+
+    /**
+     The measurement system used in spoken instructions included in route steps.
+
+     If the `includesSpokenInstructions` property is set to `true`, this property determines the units used for measuring the distance remaining until an upcoming maneuver. If the `includesSpokenInstructions` property is set to `false`, this property has no effect.
+
+     You should choose a measurement system appropriate for the current region. You can also allow the user to indicate their preferred measurement system via a setting.
+     */
+    open var distanceMeasurementSystem: MeasurementSystem = usesMetric ? .metric : .imperial
+    
+    internal static var usesMetric: Bool {
+        let locale = Locale.current as NSLocale
```

Comment:
If the `RouteOptions.locale` property is set, we should probably use that locale instead of the current one for the default value.